source/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter.h

```diff
@@ -12,7 +12,7 @@ namespace GrpcHttp1Bridge {
  */
 class Http1BridgeFilter : public Http::StreamFilter {
 public:
-  Http1BridgeFilter(Upstream::ClusterManager& cm) : cm_(cm) {}
+  Http1BridgeFilter() {}
```

Comment:
Can we kill off the empty constructors entirely here and below?

Reply:
Done.
I thought leave it along might be more readable, bad taste.

Reply:
Nah, it's just personal style.  Probably someone else on the project prefers them in as well :-P